src/Stratis.Bitcoin/P2P/Peer/NetworkPeer.cs

```diff
@@ -206,6 +247,9 @@ public bool IsConnected
         /// <inheritdoc/>
         public MessageProducer<IncomingMessage> MessageProducer { get { return this.Connection.MessageProducer; } }
 
+        /// <summary>Callback that is invoked when peer has finished disconnecting, or <c>null</c> when no notification after the disconnection is required.</summary>
+        private readonly Action<INetworkPeer> onDisconnected;
```

Comment:
No other component is using this apart for the helper right?

Reply:
Correct